README.md

```diff
@@ -29,6 +29,144 @@ This SDK supports the following features:
 * [Performance Monitoring](https://support.count.ly/hc/en-us/articles/4734457847705-Performance)
 * [Feedback Widgets](https://support.count.ly/hc/en-us/articles/4652903481753-Feedback-Surveys-NPS-and-Ratings-)
 
+## Installation
+
+In the `dependencies:` section of your `pubspec.yaml`, add the following line:
+
+```yaml
+dependencies:
+  countly_flutter: <latest_version>
```

Comment:
maybe using version would be better, but it should be maintanied

Reply:
Yeah, we could do that but the package we are trying to emulate did not.

Reply:
https://pub.dev/packages/font_awesome_flutter#installation